book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -125,10 +125,10 @@ from a file. The latter approach is handier, as it allows us to change configura
 (e.g, the window size) without having to recompile our game every time.
 
 Starting the project with `amethyst new` should have automatically generated
-`DisplayConfig` data in `config/display.ron`. If you created the
```

Comment:
It is stored in the config folder though?

Reply:
Ooh, a new commit in the tools repo. In that case, it would be nice if you could push the new version on crates.io. That should also resolve #1873 too.